{
	"schema_version": "1.6.0",
	"id": "MAL-2024-11583",
	"modified": "2024-12-09T06:50:11Z",
	"published": "2024-12-09T06:50:11Z",
	"summary": "Malicious code in eosio-signerz (PyPI)",
	"details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
	"affected": [
		{
			"package": {
				"ecosystem": "PyPI",
				"name": "eosio-signerz",
				"purl": "pkg:pypi/eosio-signerz"
			},
			"versions": [
				"0.1.0"
			],
			"database_specific": {
				"source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/eosio-signerz/MAL-2024-11583.json"
			}
		}
	],
	"credits": [
		{
			"name": "ReversingLabs",
			"contact": [
				"https://www.reversinglabs.com"
			],
			"type": "FINDER"
		}
	],
	"database_specific": {
		"malicious-packages-origins": [
			{
				"id": "RLMA-2024-11033",
				"import_time": "2024-12-09T14:38:43.442340228Z",
				"modified_time": "2024-12-09T06:50:11Z",
				"sha256": "d9a85a7be2fee3f6454ae9c75451e552f4fd0ad288cc4e524e0db624aec98806",
				"source": "reversing-labs",
				"versions": [
					"0.1.0"
				]
			}
		]
	}
}
